### ISSUE TITLE
Document how to use JavaScript functions and variables as options

### ISSUE THREAD

**Issue by cseelus** [styd/apexcharts.rb#25]
Hello again.

I guess it would be helpful if the docs state how to use JS functions, vars and statements as `options` for the chart helpers.

For example to [format labels with Apexcharts](https://apexcharts.com/docs/formatting-axes-labels/), one has to define them as a function:

```javascript
yaxis: {
  labels: {
    formatter: function (value) {
      return value + "$";
    }
  },
}
```

Also sometimes [`undefined` is needed to reset certain options](https://apexcharts.com/docs/options/xaxis/).

If you tell me where and how, I would be glad to open a PR to add that to the docs.

**Comment by styd** [comment]
Currently, it's not possible to set the formatter in options hash. I just post [an issue](https://github.com/apexcharts/apexcharts.js/issues/833) about it and propose key `prefix` and `suffix` as alternatives.
To be honest, I've never really needed to format the labels. So, I've never tried to do it. Have you used the formatter?

**Comment by cseelus** [comment]
Your proposal sounds good, I also was thinking defining it via a function seems overly verbose.

Currently we use it for one chart which shows a result with percentage sign.

**Comment by styd** [comment]
@cseelus, you're welcomed to add documentation for options hash. You can add it after all chart examples. Don't forget to mention their default values as well.

**Comment by cseelus** [comment]
You mean a reference to the [Apexcharts JS options for setting the initial configuration](https://apexcharts.com/docs/options/) or is only a subset of those options supported by Apexcharts.rb yet?

**Comment by styd** [comment]
The only options not supported currently by ApexCharts.rb are formatters. But no need to mention that on the doc.  
I'm hoping `prefix` and `suffix` will be implemented very soon. If not, I'll make  options `formatter` that accepts a closure and modify the renderer to convert the closure to JS function.

All ApexCharts.rb options should be written in snake_case instead of camelCase.

**Comment by cseelus** [comment]
A `formatter` option sounds awesome! 
How to pass properties like `undefined` to reset certain chart options?

**Comment by styd** [comment]
This documentation request has been resolved by PR #26.
